routes/error-tracker.js

```diff
@@ -72,12 +72,19 @@ function standardizeStackTrace(stackTrace) {
 }
 
 /**
- * @param {httpRequest} req
- * @param {response} res
+ * Extracts relevant information from request, handles edge cases and prepares
+ * entry object to be logged and sends it to unminification.
+ * @param {Http.Request} req
+ * @param {Http.Response} res
+ * @return { void|Promise } May return a promise that rejects on logging error
```

Comment:
Nit: Don't pad the type declaration: `{?Promise}`

Also, functions should not have conditional return. Return null otherwise.